example.zig

```diff
@@ -1,5 +1,5 @@
 const std = @import("std");
-const benchmark = @import("benchmark.zig");
+const benchmark = @import("benchmark");
```

Comment:
Could you update the code in the readme the same way?

Reply:
done